python/frameworks.html.markerb

```diff
@@ -0,0 +1,11 @@
+---
+title: Python Framework Guides
+layout: framework_docs_overview
+toc: false
+order: 8
+---
+
+
+<p>Each one of the language guides will take you through creating and deploying a simple application.</p>
+
+<p>If you don't see a framework supported here that you'd like to run on Fly.io, <%=link_to "tell us in the Community", "https://community.fly.io/" %>.</p>
```

Comment:
I'm not sure if we could sort the pages:

<img width="781" alt="Screenshot 2024-06-12 at 15 13 02" src="https://github.com/superfly/docs/assets/13651115/97ebc34c-17ad-4049-a591-e9132fdd6ca8">

I think in alphabetic order would be better:
- FastAPI
- Flask
- Streamlit

Also because FastAPI and Flask are more commonly used. [[ref](https://www.jetbrains.com/lp/devecosystem-2023/python/#python_web_libs_two_years)]